docs/devsite-help/legacy-rest-libraries.md

```diff
@@ -0,0 +1,35 @@
+# Legacy HTTP/1.1 + REST libraries
```

Comment:
If I were to use YouTube or Drive, I would be slightly scared of the legacy word here. Not certain how to address that, but maybe simply aknowledging that these are the only libraries for such APIs is fine.

Reply:
That's a good point. I think we can still keep the term in the library types doc, but I'll remove it from here. I can mention that these are effectively legacy for GCP.